proto/v1/structure/state.proto

```diff
@@ -0,0 +1,455 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "proto/v1/structure/languages.proto";
+import "proto/v1/structure/objects.proto";
+import "proto/v1/structure/versions.proto";
+
+option java_package = "org.oppia.v1.structure";
+option java_multiple_files = true;
+
+message State {
+  StateStructureVersion structure_version = 1;
+
+  SubtitledHtml content = 2;
```

Comment:
No `SubtitledHtml` message added 
`message SubtitledHtml {}` - not present

Reply:
Sorry, I don't follow. What exactly is wrong here?